gc_registry/certificate/services.py

```diff
@@ -563,8 +562,12 @@ def query_certificates(
     if validate_query(certificate_query) is False:
         return None
 
-    # Query certificates based on the given filter parameters
-    stmt = select(GranularCertificateBundle)  # type: ignore
+    # Query certificates based on the given filter parameters, without returning deleted
+    # certificates
+    stmt = select(GranularCertificateBundle).where(
+        GranularCertificateBundle.account_id == certificate_query.source_id,
+        GranularCertificateBundle.is_deleted == False,  # noqa
```

Comment:
Should this be

`~GranularCertificateBundle.is_deleted`

Reply:
Ruff seems to think so but in tariffscanner at least sqlmodel complains when passing boolean clauses through like that